OpenRA.Game/GameRules/ActorInfo.cs

```diff
@@ -18,7 +18,7 @@ namespace OpenRA
 {
 	/// <summary>
 	/// A unit/building inside the game. Every rules starts with one and adds trait to it.
-	/// Special actors like world or player are usually defined in system.yaml and affect everything.
+	/// Special actors like world or player are usually defined in world.yaml and player.yaml and affect everything.
```

Comment:
Lets please remove this line.  Engine code shouldn't be commenting about individual mod file layout choices.